### PR TITLE
Cloudfront distribution support.

### PR DESCRIPTION
**Pull request by tahir-duplo**
- `duplocloud_aws_cloudfront_distribution` resource added.

### CODE REVIEW

File: duplocloud/resource_duplo_aws_cloudfront_distribution.go [hunk 1]
```diff
@@ -0,0 +1,1984 @@
...
+					},
+					"iam_certificate_id": {
+						Type:     schema.TypeString,
+						Optional: true,
+						Computed: true,
+					},
+					"minimum_protocol_version": {
+						Type:     schema.TypeString,
+						Optional: true,
+						Default:  "TLSv1",
```
Comment (joek-duplo) [inline duplocloud/resource_duplo_aws_cloudfront_distribution.go:161]: @tahir-duplo  - we should instead use a default value that will pass compliance (`TLSv1.2_2019`)
@jeevzb  - can we do the same in the defaults for the Duplo backend / UI?

File: duplocloud/resource_duplo_aws_cloudfront_distribution.go [hunk 2]
```diff
@@ -0,0 +1,1984 @@
...
+					},
+					"ssl_support_method": {
+						Type:     schema.TypeString,
+						Optional: true,
+						Computed: true,
+						ValidateFunc: validation.StringInSlice([]string{
```
Comment (joek-duplo) [inline duplocloud/resource_duplo_aws_cloudfront_distribution.go:167]: @tahir-duplo  - can we use a default value of `sni-only` here?
@jeevzb  - can we do the same in the defaults for the Duplo backend / UI?

File: duplocloud/resource_duplo_aws_cloudfront_distribution.go [hunk 3]
```diff
@@ -0,0 +1,1984 @@
...
+								"origin_ssl_protocols": {
+									Type:     schema.TypeSet,
+									Required: true,
+									Elem: &schema.Schema{
+										Type: schema.TypeString,
+										ValidateFunc: validation.StringInSlice([]string{
+											"SSLv3",
+											"TLSv1",
+											"TLSv1.1",
+											"TLSv1.2",
```
Comment (joek-duplo) [inline duplocloud/resource_duplo_aws_cloudfront_distribution.go:334]: @tahir-duplo - we should instead use a default value that will pass compliance (`TLSv1.2`)
@jeevzb - can we do the same in the defaults for the Duplo backend / UI?

**Review by joek-duplo — CHANGES_REQUESTED** [review]

File: duplocloud/resource_duplo_aws_cloudfront_distribution.go (hunk 2 above)
Comment (tahir-duplo) [reply duplocloud/resource_duplo_aws_cloudfront_distribution.go:172]: TF changes are done.

File: duplocloud/resource_duplo_aws_cloudfront_distribution.go (hunk 3 above)
Comment (tahir-duplo) [reply duplocloud/resource_duplo_aws_cloudfront_distribution.go:340]: TF changes are done.

**Review by joek-duplo — APPROVED** [review]

**Pull request merged**